bus/accounts.go

```diff
@@ -157,8 +157,8 @@ func (a *accounts) SetBalance(id rhpv3.Account, hk types.PublicKey, balance *big
 	defer acc.mu.Unlock()
 
 	// save previous values
-	prevBalance := acc.Balance
-	prevDrift := acc.Drift
+	prevBalance := new(big.Int).Set(acc.Balance)
+	prevDrift := new(big.Int).Set(acc.Drift)
```

Comment:
was this causing issues?

Reply:
The implementation of `SetBalance` before the changes you mean? No they were not, I had a debug branch locally with logging and extra (database) fields to try and track account balance in more detail. The way it is written now is clearer though, for me at least. It's also extremely embarrassing for reasons I mentioned in DM, I'm redoing the tests on arequipa to see if drift shows up (that's why I flipped it back to `DRAFT`)